### ISSUE TITLE
[BUG] [expose] grouped windows are crashing Hyprland

### ISSUE THREAD

**Issue by Yehoward** [hyprland-community/pyprland#132]
**Pyprland version**
2.4.0

**Describe the bug**
When using `expose` command while having unlocked grouped windows on a workspace and normal windows on other workspace, the normal window will group with the grouped ones and may crash the system.

**To Reproduce**
1. Group windows on a workspace and spawn a normal window on other workspace
2. Call them with `expose`

I think the best option here is to lock all the grouped windows before calling them

**Comment by fdev31** [comment]
Using the `lockgroups` dispatcher  with `lock` or `unlock` seem straightforward.
@Yehoward did you test locked groups were indeed surviving the expose operation?

**Comment by Yehoward** [comment]
Just tested, the locked groups also seem to crash now, on hyprland 0.41.2-1.

**Comment by fdev31** [comment]
So, when you say crash it's a real crash? I can't even test this feature because Hyprland is crashing...
Need to check which IPC commands are making it crash and open a bug there.

**Comment by fdev31** [comment]
This is not a Pyprland bug, but I can help with the Hyprland bug eventually.

**Comment by fdev31** [comment]
I don't think this is still happening... at least it works for me. Re-open if still facing the issue.
